app/scripts/constants.js

```diff
@@ -555,5 +555,10 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
     //   href: 'http://example.com/',
     //   tooltip: 'Open Dashboard'
     // }
-  ]
+  ],
+  QUOTA_NOTIFICATION_MESSAGE: {
```

Comment:
Did we intend to leave this in as an empty object?

Reply:
I believe so.  Similar to the other two empty objects right before this one.

Reply:
Yeah, they're for custom messages. We generate a default message in code.